subcommands/plot_run_comp.qmd

```diff
@@ -20,7 +20,7 @@ pyani-plus plot-run [OPTIONS]
 
 `--outdir`:  Output directory. (<span style="color:green;">DIRECTORY</span>)  [<span style="color:red;">REQUIRED</span>]
 
-`--run-ids`: Which runs (comma separated list, reference first)? (<span style="color:green;">TEXT</span>) [<span style="color:red;">REQUIRED</span>]
+`--run-ids`: Which runs (comma separated list, reference first)? Ex. 1,2,3,4 (reference = 1) (<span style="color:green;">TEXT</span>) [<span style="color:red;">REQUIRED</span>]
```

Comment:
In this meant to be a copy-and-paste of the command line help? If so, it should stay in sync. If not, we can be more verbose here.